Gordon360/Controllers/EmailsController.cs

```diff
@@ -3,6 +3,7 @@
 using Gordon360.Models.ViewModels;
 using Gordon360.Services;
 using Gordon360.Static.Names;
+using Microsoft.AspNetCore.Authorization;
```

Comment:
Why are you adding this here?

Reply:
You are right! I modified this then realized I wasn't in the right file, but I didn't change it back.